src/main/app/forms/models/housingDisrepair.ts

```diff
@@ -0,0 +1,66 @@
+import { IsDefined, ValidateIf, IsIn } from 'class-validator'
+import { Serializable } from 'models/serializable'
+import { YesNo } from 'forms/models/yesNo'
+import { GeneralDamages } from 'forms/models/generalDamages'
+
+export class ValidationErrors {
+  static readonly HOUSING_DISREPAIR_REQUIRED: string = 'Please select yes or no'
+  static readonly GENERAL_DAMAGES_REQUIRED: string = 'Please select more or less'
+  static readonly OTHER_DAMAGES_REQUIRED: string = 'Please select how much you expect to recover'
+}
+
+export class OtherDamages {
+  static readonly LESS: string = 'less'
+  static readonly MORE: string = 'more'
+  static readonly NONE: string = 'none'
+
+  static all (): string[] {
+    return [
+      OtherDamages.LESS,
+      OtherDamages.MORE,
+      OtherDamages.NONE
+    ]
+  }
+}
+
+export class HousingDisrepair implements Serializable<HousingDisrepair> {
+
+  @IsDefined({ message: ValidationErrors.HOUSING_DISREPAIR_REQUIRED })
+  @IsIn(YesNo.all(), { message: ValidationErrors.HOUSING_DISREPAIR_REQUIRED })
+  housingDisrepair?: string
```

Comment:
I would recommend using types here instead of string, would need a `fromObject` function see https://git.reform.hmcts.net/cmc/citizen-frontend/blob/master/src/main/features/response/form/models/response.ts#L11
https://git.reform.hmcts.net/cmc/citizen-frontend/blob/master/src/main/features/response/form/models/responseType.ts